src/Models/ExternalLink.php

```diff
@@ -27,8 +30,13 @@ class ExternalLink extends Link
     public function getCMSFields(): FieldList
     {
         $this->beforeUpdateCMSFields(function (FieldList $fields) {
-            $linkField = $fields->dataFieldByName('ExternalUrl');
-            $linkField->setTitle(_t(__CLASS__ . '.EXTERNAL_URL_FIELD', 'External url'));
+            $field = UrlField::create('ExternalUrl');
+            $field->setTitle(_t(__CLASS__ . '.EXTERNAL_URL_FIELD', 'External URL'));
```

Comment:
```suggestion
            $field = UrlField::create('ExternalUrl', $this->fieldLabel('ExternalUrl'));
```
This should have been done like this originally, since the field label localisation is provided (see `db_ExternalUrl`) - but I won't block merging if you don't make this change since it's not directly related to what this PR is for.

If you do make this change, please make it for other link types as well.

Reply:
Leave as is